homeassistant/components/zwave_js/__init__.py

```diff
@@ -36,16 +38,21 @@ def register_node_in_dev_reg(
         config_entry_id=entry.entry_id,
         identifiers={(DOMAIN, client.driver.controller.home_id, node.node_id)},
         sw_version=node.firmware_version,
-        name=node.name,
-        model=node.product_type,
-        manufacturer=str(node.manufacturer_id),
+        name=node.name or node.device_config.description,
```

Comment:
Not all nodes have a device config.

Reply:
I've made that into a class so it will return None and not break

Reply:
as a last resort we can perhaps resort to a value that is always there such as "Node X"

Reply:
nvm, it works as expected. If needed we can extend this to do some more fallback names but it will not crash.